thehive-charts/thehive/values.yaml

```
@@ -2,47 +2,59 @@
 # TheHive Helm Chart variables #
 ################################
 
+# Refer to [this documentation page](https://docs.strangebee.com/thehive/installation/kubernetes/) for installation instructions and configuration.
+
+# -- Override the name of the chart
 nameOverride: ""
+# -- Override the full name of the chart
 fullnameOverride: ""
 
+# -- Image pull secrets for private registries
 imagePullSecrets: []
 #  - name: "image-pull-secret"
 
 thehive:
-  # Additional labels and annotations to attach to TheHive resources
-  # (dedicated keys exist for Ingress and ServiceAccount annotations)
+  # -- Additional labels to attach to TheHive resources
   labels: {}
+  # -- Additional annotations to attach to TheHive resources
   annotations: {}
 
-  # Additional labels and annotations to attach to TheHive pods
+  # -- Additional labels to attach to TheHive pods
   podLabels: {}
+  # -- Additional annotations to attach to TheHive pods
   podAnnotations: {}
 
-  # Only used when thehive.autoscaling.enabled is set to false
+  # -- Number of TheHive replicas (only used when autoscaling.enabled is false)
   replicas: 1
 
-  # TheHive Deployment strategy
   strategy:
-    # Set to "RollingUpdate" or "Recreate" depending on HA needs and TheHive version upgrade (schema migrations)
+    # -- Deployment strategy type. Set to "RollingUpdate" or "Recreate" depending on HA needs and TheHive version upgrade (schema migrations)
     type: RollingUpdate
-    # Only used for "RollingUpdate" strategy type
     rollingUpdate:
+      # -- Maximum number of pods that can be created over the desired number of pods (only for RollingUpdate)
       maxSurge: 1
+      # -- Maximum number of pods that can be unavailable during the update (only for RollingUpdate)
       maxUnavailable: 1
 
-  # PodDisruptionBudget configuration
+  # -- Maximum number of unavailable pods in PodDisruptionBudget
   maxUnavailable: 1
 
-  # TheHive nodes count
+  # -- Minimum number of TheHive nodes in cluster
```

```suggestion
  # -- Minimum number of TheHive nodes to form a cluster
```
Taken from [TheHive doc](https://docs.strangebee.com/thehive/configuration/thehive-docker-entrypoint-settings/#clustering-configuration)